evap/staff/tests.py

```diff
@@ -304,16 +304,11 @@ def test_all_urls(self):
             ("test_staff_semester_x_import", "/staff/semester/1/import", "evap"),
             ("test_staff_semester_x_assign", "/staff/semester/1/assign", "evap"),
             ("test_staff_semester_x_lottery", "/staff/semester/1/lottery", "evap"),
-            ("test_staff_semester_x_reset", "/staff/semester/1/reset", "evap"),
-            ("test_staff_semester_x_contributorready", "/staff/semester/1/contributorready", "evap"),
-            ("test_staff_semester_x_approve", "/staff/semester/1/approve", "evap"),
-            ("test_staff_semester_x_publish", "/staff/semester/1/publish", "evap"),
             # staff semester course
             ("test_staff_semester_x_course_y_edit", "/staff/semester/1/course/5/edit", "evap"),
             ("test_staff_semester_x_course_y_email", "/staff/semester/1/course/1/email", "evap"),
             ("test_staff_semester_x_course_y_preview", "/staff/semester/1/course/1/preview", "evap"),
             ("test_staff_semester_x_course_y_comments", "/staff/semester/1/course/5/comments", "evap"),
-            ("test_staff_semester_x_course_y_unpublish", "/staff/semester/1/course/8/unpublish", "evap"),
             ("test_staff_semester_x_course_y_delete", "/staff/semester/1/course/1/delete", "evap"),
             # staff questionnaires
             ("test_staff_questionnaire", "/staff/questionnaire/", "evap"),
```

Comment:
we should decide whether we want to have this kind of test and then either keep the url list complete or remove it altogether. i vote for the former. if you agree, iirc there was a PR of yours that also did not add a new url, please find that out and add it as well.